app/services/usps_in_person_proofing/enrollment_helper.rb

```diff
@@ -1,61 +1,94 @@
 module UspsInPersonProofing
   class EnrollmentHelper
-    def save_in_person_enrollment(user, profile, pii, selected_location_details = nil)
-      enrollment = InPersonEnrollment.create!(
-        profile: profile,
-        user: user,
-        current_address_matches_id: pii['same_address_as_id'],
-        selected_location_details: selected_location_details,
-      )
-
-      enrollment_code = create_usps_enrollment(enrollment, pii)
-      return unless enrollment_code
-
-      # update the enrollment to status pending
-      enrollment.enrollment_code = enrollment_code
-      enrollment.status = :pending
-      enrollment.save!
-
-      send_ready_to_verify_email(user, pii, enrollment)
-    end
+    class << self
+      def schedule_in_person_enrollment(user, profile, pii, selected_location_details = nil)
+        enrollment = user.establishing_in_person_enrollment ||
+                     InPersonEnrollment.create!(user: user)
+
+        enrollment.profile = profile
+        enrollment.current_address_matches_id = pii['same_address_as_id']
+        if enrollment.selected_location_details.blank?
+          enrollment.selected_location_details = selected_location_details
+        end
```

Comment:
Do we still need to support the argument? Could we always rely on the enrollment record having already assigned it?